docs/reference/support-policy.md

```diff
@@ -104,20 +104,22 @@ compatibility tables to describe the interoperability requirements of these piec
 
 ### JavaScript Runtime
 
-| Stencil Version | Node v10 | Node v12 | Node v14 | Node v16 | Node v18  | Node v20 |  Deno*  |
-|:---------------:|:--------:|:--------:|:--------:|:--------:|:---------:|:--------:|:-------:|
-|       V4        | &#10060; | &#10060; | &#10060; | &#9989;  |  &#9989;  | &#9989;  | &#9888; |
-|       V3        | &#10060; | &#10060; | &#9989;  | &#9989;  |  &#9989;  | &#9989;  | &#9888; |
-|       V2        | &#10060; | &#9989;  | &#9989;  | &#9989;  | &#9888;** | &#10060; | &#9888; |
-|       V1        | &#9989;  | &#9989;  | &#9989;  | &#9989;  | &#10060;  | &#10060; | &#9888; |
-
-\* Experimental Deno support was available in Stencil
+| Stencil Version | Node v10 | Node v12 | Node v14 | Node v16 | Node v18 | Node v20 |
+|:---------------:|:--------:|:--------:|:--------:|:--------:|:--------:|:--------:|
+|       V4        | &#10060; | &#10060; | &#10060; | &#9989;  | &#9989;  | &#9989;  |
+|       V3        | &#10060; | &#10060; | &#9989;  | &#9989;  | &#9989;  | &#9989;  |
+|       V2        | &#10060; | &#9989;  | &#9989;  | &#9989;  | &#9888;* | &#10060; |
+|       V1        | &#9989;  | &#9989;  | &#9989;  | &#9989;  | &#10060; | &#10060; |
+
+\* Node 18 is likely to work with Stencil v2, although it was never formally supported
+
+:::note
+Experimental Deno support was available in Stencil
 [v1.16.0](https://github.com/ionic-team/stencil/releases/tag/v1.16.0) through
 [v2.8.1](https://github.com/ionic-team/stencil/releases/tag/v2.8.1). This experimental support was removed in
 [v2.9.0](https://github.com/ionic-team/stencil/releases/tag/v2.9.0). For additional reasoning behind this decision,
 please see [this document](https://github.com/ionic-team/stencil/blob/main/docs/adr/0013-deno-removal.md).
-
-\** Node 18 is likely to work with Stencil v2, although it was never formally supported
+:::
```

Comment:
just a question - do we even need to leave this in? it was never widely used and we took it out quite a while ago now

Reply:
Yeah, I considered that myself and simply didn't have a super strong opinion - removed in https://github.com/ionic-team/stencil-site/pull/1125/commits/e6d13441844bff8bda05ed07da736f84963b56ed

Reply:
oh sorry should have been more clear - meant the whole `:::note` block, both the node 18 bit and the stuff about Deno

Reply:
Ah sorry, I didn't read the original comment as carefully as I thought the first time around. Yeah...I can get on board with removing it - aa078616c 

Tagging in @tanner-reits again to re-review since things have changed (and he originally brought up removing the column a couple weeks ago)